gazebo_plugins/include/gazebo_plugins/gazebo_ros_gpu_laser.h

```diff
@@ -82,6 +82,12 @@ namespace gazebo
     /// \brief for setting ROS name space
     private: std::string robot_namespace_;
 
+    /// \brief time between measurement points (Laserscan ROS msg property)
+    private: float time_increment_;
+
+    /// \brief time in between scans (Laserscan ROS msg property)
+    private: float scan_time_;
```

Comment:
Adding new private member breaks ABI. We sometimes let it go through in this repo because certain plugins are not often inherited, but I suspect these may be part of those which are often extended.

I'm not sure what's the best way around this. Maybe get the increment and calculate the scan time at every `OnScan` callback?

Any thoughts, @j-rivero ?

Reply:
note: this also holds for the `time_increment` member

Reply:
Correct, it applies to both of them.